src/sentry/static/sentry/app/views/eventsV2/table/columnEditRow.tsx

```diff
@@ -331,12 +332,19 @@ class ColumnEditRow extends React.Component<Props> {
   }
 
   render() {
-    const {className, takeFocus, gridColumns} = this.props;
+    const {className, takeFocus, gridColumns, showFunctionsOnly} = this.props;
     const {field, fieldOptions, parameterDescriptions} = this.getFieldData();
 
+    let selectFields = Object.values(fieldOptions);
+    if (showFunctionsOnly) {
+      selectFields = selectFields.filter(
+        selectField => selectField.value.kind === FieldValueKind.FUNCTION
+      );
+    }
+
```

Comment:
This is the bit filtering out anything that's not a function from the first selector.